go/pkg/rexec/rexec_test.go

```diff
@@ -36,8 +36,10 @@ func TestExecCacheHit(t *testing.T) {
 	res, meta := e.Client.Run(context.Background(), cmd, opt, oe)
 
 	wantMeta := &command.Metadata{
-		CommandDigest: cmdDg,
-		ActionDigest:  acDg,
+		CommandDigest:    cmdDg,
```

Comment:
Can we also have a test where the input file bytes are not zero?

Reply:
Sure, added an input file to one of the tests.